qemu/tests/virtio_serial_file_transfer.py

```diff
@@ -191,10 +219,13 @@ def transfer_data(params, vm, host_file_name=None, guest_file_name=None,
     guest_cmd = ("%s %s -d %s -f %s -a %s" %
                  (python_bin, guest_script,
                   port_name, guest_file_name, guest_action))
+    kill_pid()
     result = _transfer_data(session, host_cmd, guest_cmd, transfer_timeout, sender)
     clean_cmd = params['clean_cmd']
-    for script in guest_scripts.split(";"):
-        session.cmd('%s %s' % (clean_cmd, script))
+    process.system('rm -f %s' % host_file_name)
```

Comment:
same as above, please use functions from `Python` standard modules. In this case, why not use `os.remove` or `os.unlink`